src/internal/connector/exchange/service_restore.go

```diff
@@ -425,10 +425,12 @@ func restoreCollection(
 	}
 }
 
-// generateRestoreContainerFunc utility function that holds logic for creating
-// Root Directory or necessary functions based on path.CategoryType
-// Assumption: collisionPolicy == COPY
-func GetContainerIDFromCache(
+// MakeContainerDestinaion builds the destination into the container
+// at the provided path.  As a precondition, the destination cannot
+// already exist.  If it does then an error is returned.  The provided
+// containerResolver is updated with the new destination.
+// @ returns the container ID of the new destination container.
+func MakeContainerDestinaion(
```

Comment:
This name should be updated.  I would suggest using `Create` over `Make` in the naming convention. Be sure to update the name through out the PR.